nengo/neurons.py

```diff
@@ -215,7 +219,24 @@ def rates(self, x, gain, bias):
         self.step_math(dt=1.0, J=J, output=out)
         return out
 
-    def step_math(self, dt, J, output):
+    def make_neuron_state(self, phases, dt, dtype=None):
+        states = [p for p in self.probeable if p not in ("rates", "spikes")]
+
+        result = OrderedDict((state, 0) for state in states)
+        if "voltage" in result:
+            # apply phases to voltages
+            result["voltage"] = phases
+
+        return result
+
+        # # distribute phases equally across states (each neuron will be at a random
+        # # position between 0 and 1 in one of the states, and 0 in all other states)
+        # phase_frac, phase_int = np.modf(phases * len(states))
+        # return OrderedDict(
+        #     (state, (phase_int == i) * phase_frac) for i, state in enumerate(states)
+        # )
```

Comment:
I realized why this code to also set the initial refractory period wasn't working: the refractory period has range [0, tau_ref], not [0, 1]. So to initialize it, we'd have to know `tau_ref`.